src/redux/rockets/rockesSlice.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
```

Comment:
- [ ] Again, let us fix the eslint error without disable line of code, kindly find another way of correcting the eslint lint errors